### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

File: docs/async_message_producer.md [hunk 1]
```diff
@@ -1,61 +1,65 @@
...
-
-## Async Querying
-
-An async message producer for the QueryBus needs to provide a response by resolving the handed over `React\Promise\Deferred`.
-When using a messaging system like ZeroMQ for example you can make use of request/response mode or RPC mode.
-HTTP APIs provide responses naturally.
-So these are both good candidates to use for remote querying.
+# Async Message Producer
+
+Messaging becomes really interesting if you process your messages asynchronous. PSB can
```
Comment (lucascourot) [inline docs/async_message_producer.md]: asynchronously

File: docs/async_message_producer.md [hunk 2]
```diff
@@ -1,61 +1,65 @@
...
+message producer which hands them over to a messaging system like rabbitMQ, zeroMQ, gearman or beanstalkd.
+
+## Available MessageProducer
+
+- [BernardProducer](https://github.com/prooph/psb-bernard-producer): Queue multi-backend providing different
+  drivers like Doctrine DBAL and Predis (see
+  [https://github.com/bernardphp/bernard](https://github.com/bernardphp/bernard) for a complete list of drivers)
+- [GuzzleHttpProducer](https://github.com/prooph/psb-http-producer): Send messages to a remote system using
+  HTTP
+- [ZeromqProducer](https://github.com/prooph/psb-zeromq-producer): Async message handling using super fast
```
Comment (bweston92) [inline docs/async_message_producer.md]: Keeping consistency with the name within the package you're linking too.

```
- ZeromqProducer
+ ZeroMQProducer
```

File: docs/async_message_producer.md [hunk 3]
```diff
@@ -1,61 +1,65 @@
...
-When using a messaging system like ZeroMQ for example you can make use of request/response mode or RPC mode.
-HTTP APIs provide responses naturally.
-So these are both good candidates to use for remote querying.
+# Async Message Producer
+
+Messaging becomes really interesting if you process your messages asynchronous. PSB can
+hide such an asynchronous workflow behind a unified interface. You can start with synchronous message
+dispatching by routing your messages directly to message handlers and if you later want to improve response
+times you can switch to async processing on a per message basis by routing the appropriate messages to a
+message producer which hands them over to a messaging system like rabbitMQ, zeroMQ, gearman or beanstalkd.
```
Comment (bweston92) [inline docs/async_message_producer.md]: Capitalise names.

```
- rabbitMQ, zeroMQ, gearman or beanstalkd.
+ RabbitMQ, ZeroMQ, Gearman, Beanstalkd or any other queue.
```

**Comment by prolic** [comment]
/cc @sandrokeil @codeliner 

it uses now a custom prooph/bookdown-template (not much changed, added logo and intend for chapters)

**Pull request merged**